### PR TITLE
BACKLOG-12037 improve style of CE type selector

### PR DESCRIPTION
**Pull request by Taybou**

### CODE REVIEW

File: src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx [hunk 1]
```diff
@@ -66,36 +71,36 @@ const CreateNewContentDialogCmp = ({open, parentPath, onExited, onClose, onCreat
...
+                        if (!node.nodeType.mixin) {
+                            onCreateContent(node);
+                        }
+                    }}
+                />
             </div>
             <DialogActions>
                 <Button variant="secondary" data-sel-role="content-type-dialog-cancel" onClick={onClose}>
-                    {t('content-editor:label.contentEditor.CMMActions.createNewContent.btnDiscard')}
+                    {t('content-editor:label.contentEditor.CMMActions.createNewContent.btnDiscard').toUpperCase()}
```
Comment (dgriffon) [inline src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx]: Same here

File: src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx [hunk 2]
```diff
@@ -66,36 +71,36 @@ const CreateNewContentDialogCmp = ({open, parentPath, onExited, onClose, onCreat
...
                     disabled={!selectedType}
                     variant="primary"
                     onClick={() => {
-                            onCreateContent(selectedType);
-                        }}
+                        onCreateContent(selectedType);
+                    }}
                 >
-                    {t('content-editor:label.contentEditor.CMMActions.createNewContent.btnCreate')}
+                    {t('content-editor:label.contentEditor.CMMActions.createNewContent.btnCreate').toUpperCase()}
```
Comment (dgriffon) [inline src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx]: same here

File: src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx [hunk 3]
```diff
@@ -51,13 +55,14 @@ const CreateNewContentDialogCmp = ({open, parentPath, onExited, onClose, onCreat
     const filteredTree = filterTree(data.forms.contentTypesAsTree, selectedType, filter);
     return (
         <Dialog open={open} aria-labelledby="dialog-createNewContent" onExited={onExited} onClose={onClose}>
-            <DialogTitle id="dialog-createNewContent">
+            <DialogTitle className={classes.dialogTitle} id="dialog-createNewContent">
                 <Typography color="alpha" variant="delta">
-                    {t('content-editor:label.contentEditor.CMMActions.createNewContent.labelModal')}
+                    {t('content-editor:label.contentEditor.CMMActions.createNewContent.labelModal').toUpperCase()}
```
[the remark this answers is not in the document]
Comment (Taybou) [reply src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx]: If we want to reuse the label later.

File: src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx (hunk 1 above)
Comment (Taybou) [reply src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx]: If we want to reuse the label later.

File: src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx (hunk 2 above)
Comment (Taybou) [reply src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx]: If we want to reuse the label later.

File: src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (dgriffon) [reply src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx]: Then we will use a new key for the lowercase value.

[the remark this answers is not in the document]
Comment (Taybou) [reply src/javascript/Create/CreateNewContentAction/CreateNewContentDialog.jsx]: All right, thank you

**Pull request merged**